### PR TITLE
tweaks for travis

### PR DESCRIPTION
**Pull request by doutriaux1**
@aashish24 @dlonie just testing please don't even bother looking at it

### CODE REVIEW

**Comment by durack1** [comment]
@doutriaux1 another one of these weird timeouts again [LINK](https://travis-ci.org/UV-CDAT/uvcdat/builds/53841592).. Who knows if there was a problem or not..

**Comment by durack1** [comment]
@doutriaux1 this should probably go into the default `.travis.yml` file:

http://docs.travis-ci.com/user/build-timeouts/#Build-times-out-because-no-output-was-received

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,6 @@ script:
     - cd ..
     - mkdir _build
     - cd _build
-    - cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=LEAN -DCDAT_BUILD_GRAPHICS=ON -DCDAT_BUILD_SCIPY=OFF ../uvcdat
+    - cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=DEFAULT -DCDAT_BUILD_GUI=OFF -DCDAT_BUILD_SCIPY=OFF ../uvcdat
```
Comment (aashish24) [inline .travis.yml]: Given the time on travis, we can only build in LEAD mode or it will fail most of the time.

**Comment by durack1** [comment]
@aashish24 I don't think this is true, with travis ci we have 120 mins.. It also seems that the problem is more related to timeouts within the install (no activity for 20 mins) rather than exceeding the 120 min limit

**Comment by aashish24** [comment]
@durack1  please see below (http://docs.travis-ci.com/user/build-configuration). In any case, I want to keep the build process shorted so that we can provide quick feedback. Like I said, I am working on another solution until then, please don't change the travis-ci. 

----

Because it is very common to see test suites or before scripts to hang up, Travis CI has hard time limits. If a script or test suite takes longer to run, the job will be forcefully terminated and you will see a message about this in your build log.

With our current timeouts, a job will be terminated if it's still running after 50 minutes (120 minutes on travis-ci.com), or if there hasn't been any log output in 10 minutes.

Some common reasons why test suites may hang up:

Waiting for keyboard input or other kind of human interaction
Concurrency issues (deadlocks, livelocks and so on)
Installation of native extensions that take very long time to compile
There is no such timeout for a build; a build will run as long as all the jobs do.

**Comment by durack1** [comment]
@aashish24 the issue was opened as it appeared the travis config (rather than any other change) was causing the build failures.. It seems fairly useless to have tests that are run and are failing if the only part failing is travis itself, rather than due to changes in the PR request being tested.

In #1126 in particular this is not even built under `LEAN` conditions, so it just shouldn't fail

**Comment by doutriaux1** [comment]
@aashish24 the first comment says to you to not worry about this one. This is just @durack1 trying out things, when he's ready we will ask you to review don't worry. Re-opening so that the changes he makes get tested on travis.

**Comment by aashish24** [comment]
things are tested on travis whether on not it has a pull request. Travis test every commit. Closing it for that reason.

**Pull request closed without merging**